README.md

```diff
@@ -1 +1,2 @@
-### Ah the phoenix
\ No newline at end of file
+### Ah the phoenix
+[![Build Status](https://travis-ci.com/andela/ah-the-phoenix-frontend.svg?branch=develop)](https://travis-ci.com/andela/ah-the-phoenix-frontend)
```

Comment:
Hi @Arusey I think the badges should be placed at the very top of the readme, just before anything else is typed.

Reply:
Noted